doc/extdev/parserapi.rst

```diff
@@ -3,6 +3,35 @@
 Parser API
 ==========
 
+`The document of docutils describes`__ about the parsers like following:
+
+    The Parser analyzes the the input document and creates a node tree
+    representation.
+
+__ http://docutils.sourceforge.net/docs/dev/hacking.html#parsing-the-document
+
+In Sphinx, the parser modules works as same as docutils.  The parsers are
+registered to Sphinx by extensions using Application APIs;
+:meth:`Sphinx.add_source_suffix()` and :meth:`Sphinx.add_source_parsers()`.
+
+The source_suffix is a mapping from file suffix to file type.  For example,
```

Comment:
s/source_suffix/:confval:`source_suffix`/ ?

Reply:
That's fair. I'd probably remove the underscore in that case and make it italic so it stands out:

    *source suffix*

Otherwise though, this is fine

Reply:
Sounds good.